transport/src/main/java/io/netty/channel/embedded/EmbeddedChannelId.java

```diff
@@ -40,7 +40,7 @@ public String asLongText() {
     }
 
     @Override
-    public int compareTo(ChannelId o) {
+    public int compareTo(final ChannelId o) {
```

Comment:
The goal of the patch was to clean up equals/compareTo/hashCode methods on the `ChannelId` implementations. :smile: